astroid/rebuilder.py

```diff
@@ -873,8 +876,12 @@ def visit_arguments(self, node: "ast.arguments", parent: NodeNG) -> nodes.Argume
         )
         # save argument names in locals:
         if vararg:
+            if not newnode.parent:
```

Comment:
`Mypy` doesn't recognise:

```python
if vararg or kwarg and not new node.parent:
    raise ParentMissingError(target=newnode)
```

Reply:
Shouldn't `ast.argument` nodes always have a parent anyway?
Why not do:
```py
if not newnode.parent:
    raise ParentMissingError(target=newnode)
```

Reply:
I thought so but didn't want to make any false assumptions. This was the safer option. I'll add the error.